### DISCUSSION TITLE
`Analyst` can't generate CSV reports

### DISCUSSION THREAD

**Discussion by mariuskarotkis** [Security-Onion-Solutions/securityonion#10738] [Unsupported Versions]
Version: 2.3.260

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/3322571/6196f2fb-620f-4afc-b63c-a0cbf2c10a18)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/3322571/94a7bb2c-e9f5-4be1-8769-697f89cbd092)

```
Error: Forbidden
    at fetch_Fetch.fetchResponse (details masked)
```

**Comment by cm-ops** [comment]
Try adding the reporting_user role - `sudo so-user addrole "USERNAME@DOMAIN" reporting_user` and see if you can generate the CSV.

**Reply by mariuskarotkis** [reply]
It works. Thanks.
